docs/release-notes/cloud/cloud-2024-09-adding-eslint9-and-pmd7.md

```diff
@@ -0,0 +1,30 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Adding ESLint 9 and PMD 7 as new supported tools September, 2024
+
+On September, 2024 Codacy is adding ESLint 9 and PMD 7 as new supported tools and it will deprecate ESLint 8 and PMD 6 soon:
+
+-   ESLint 9 and PMD 7 won't be enabled by default on new repositories for now and Codacy recommends that you migrate to this version of the tool to benefit from the new features and fixes of ESLint and PMD
```

Comment:
…won’t be enabled by default for new repositories. 

Update these tools version to benefit from the new features and fixes.